src/uwtools/config/formats/base.py

```diff
@@ -25,8 +23,6 @@ class Config(ABC, UserDict):
 
     def __init__(self, config: Optional[Union[dict, str, Path]] = None) -> None:
         """
-        Construct a Config object.
```

Comment:
In a number of places here and below, docstring content that looked fine just in code looked odd, incomplete, etc. when rendered into RST docs. Be sure to review the rendered documentation in Read the Docs to see how all of this looks.